source/config/__init__.py

```diff
@@ -65,6 +65,7 @@ def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	#possible log levels are DEBUG, IO, DEBUGWARNING, INFO
 	loggingLevel = string(default="INFO")
 	showWelcomeDialogAtStartup = boolean(default=true)
+	showSpeechViewerAtStartup = boolean(default=false)

```

Comment:
What about a sub-section under general?


On 7/21/2016 9:54 PM, Michael Curran wrote:
>
> In source/config/__init__.py 
> <https://github.com/nvaccess/nvda/pull/6197#discussion_r71823659>:
>
> > @@ -65,6 +65,7 @@ def validateConfig(configObj,validator,validationResult=None,keyList=None):
> >  	#possible log levels are DEBUG, IO, DEBUGWARNING, INFO
> >  	loggingLevel = string(default="INFO")
> >  	showWelcomeDialogAtStartup = boolean(default=true)
> > +	showSpeechViewerAtStartup = boolean(default=false)
>
> Perhaps move this option into a separate speechViewer section, as we 
> are planning to also save position information as well.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6197/files/0581ada05316383e7903e30dffc09f1256d9a3ce#r71823659>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivVg07p9-WQGZPgeeSQLugNHBj2EOks5qYD7ggaJpZM4JRhJV>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194